Adding-new-translations-for-i18n.md

```diff
@@ -1,14 +1,33 @@
+## Quick overview
 
-## Contributing translations to Oppia
+In order to ensure that the Oppia website is understandable to learners around the world, we provide internationalization (i18n) support on Oppia for learner-facing pages. This enables learners to view the site in different languages using the language-selector dropdown in the navbar.
 
-Thanks to translatewiki.net, it's easy to add or update translations for Oppia! To do so, just follow these steps:
+Our i18n support uses the [angular-translate](http://angular-translate.github.io/) library, documentation for which can be found [here](https://angular-translate.github.io/docs/#/guide). All platform string translations are provided through our partner, [translatewiki.net](https://translatewiki.net/wiki/Translating:Oppia).
 
-1. Visit the [Oppia Translatewiki](https://translatewiki.net/wiki/Translating:Oppia) page.
+This wiki page explains how to:
+- [Fill in missing platform translations](#how-to-fill-in-missing-platform-translations)
+- [Translate lessons and other dynamic text](#how-to-translate-lessons-and-other-dynamic-text)
+- [Create an i18n-compliant PR](#how-to-create-an-i18n-compliant-pr)
+- [Fetch new translations from translatewiki](#how-to-fetch-new-translations-from-translatewiki)
+- [Add a new translation language](#how-to-add-a-new-translation-language)
+
+## How to fill in missing platform translations
+
+All our translations are contributed through translatewiki.net. If you see missing translations in a language you're familiar with, please follow these steps to help fix the gap (do not create a PR):
+
+1. Visit the [Oppia Translatewiki](https://translatewiki.net/wiki/Translating:Oppia) page. Read the notes on that page regarding plural rules and special markup syntax.
 2. Click "Translate this project".
 3. Select a language to contribute translations for. (Also, read the [notes](https://github.com/oppia/oppia/wiki/Adding-new-translations-for-i18n#note-1-variable-replacement) below describing the translation formats used for variables and plurals.)
 
-Changes will then be pushed to Oppia automatically by the translatewiki admins (**@Nikerabbit** and **@siebrand**), and they will show up in future Oppia releases.
+Changes will then be pushed to Oppia automatically by the Translatewiki admins, and they will show up in future Oppia releases. We typically update new translations to the Oppia.org website on a monthly cadence.
+
+### Important: Don't rely on machine translation
+
+The translatewiki admins have requested that translators do not rely on machine auto-translation, especially if they don't know the language and cannot fix its mistakes. 
+
+In cases where filling in a translation is necessary for technical reasons, they recommend doing the following:
+- Add the string `!!FUZZY!!` to the beginning of the machine-translated string. This will update the translation in Oppia's source tree, and also mark it as "needing update" for the translators on translatewiki.
```

Comment:
Do we have technical safeguards to block these fuzzy-labeled translations from being shown to users in the time before they get manually translated?

Reply:
We don't, but that's OK IMO. A not-so-great translation is better than no translation.